buildlib/dockers/docker-compose.yml

```diff
@@ -0,0 +1,89 @@
+version: "3"
+
+services:
+  centos7-mofed5.1-cuda11.1:
+    image: centos7-mofed5.1-cuda11.1
+    build:
+      context: .
+      network: host
+      dockerfile: centos-release.Dockerfile
+      args:
+        MOFED_VERSION: 5.1-2.5.8.0
+        MOFED_OS: rhel7.6
+        CUDA_VERSION: 11.1
+        OS_VERSION: 7
+  centos8-mofed5.1-cuda11.1:
+    image: centos8-mofed5.1-cuda11.1
+    build:
+      context: .
+      network: host
+      dockerfile: centos8-release.Dockerfile
+      args:
+        MOFED_VERSION: 5.1-2.5.8.0
+        MOFED_OS: rhel8.3
+        CUDA_VERSION: 11.1
+        OS_VERSION: 8
+  ubuntu18.04-mofed5.1-cuda11.1:
+    image: ubuntu18.04-mofed5.1-cuda11.1
+    build:
+      context: .
+      network: host
+      dockerfile: ubuntu-release.Dockerfile
+      args:
+        MOFED_VERSION: 5.1-2.5.8.0
+        UBUNTU_VERSION: 18.04
+        CUDA_VERSION: 11.1
+  ubuntu20.04-mofed5.1-cuda11.1:
+    image: ubuntu20.04-mofed5.1-cuda11.1
+    build:
+      context: .
+      network: host
+      dockerfile: ubuntu-release.Dockerfile
+      args:
+        MOFED_VERSION: 5.1-2.5.8.0
+        UBUNTU_VERSION: 20.04
+        CUDA_VERSION: 11.1
+  ubuntu20.10-mofed5.1-cuda11.1:
+    image: ubuntu20.10-mofed5.1-cuda11.1
+    build:
+      context: .
+      network: host
+      dockerfile: ubuntu-release.Dockerfile
+      args:
+        MOFED_VERSION: 5.1-2.5.8.0
+        UBUNTU_VERSION: 20.10
+        CUDA_VERSION: 11.1
+        MOFED_OS: ubuntu20.04
+  fedora31-mofed5.1-cuda11.1:
+    image: fedora31-mofed5.1-cuda11.1
+    build:
+      context: .
+      network: host
+      dockerfile: fedora-release.Dockerfile
+      args:
+        MOFED_VERSION: 5.1-2.5.8.0
+        OS_VERSION: 31
+        CUDA_VERSION: 11.1
+        MOFED_OS: fc31
+  # sles12.5-mofed5.1-cuda11.1:
+  #   image: sles11-mofed5.1-cuda11.1
+  #   build:
+  #     context: .
+  #     network: host
+  #     dockerfile: sles-release.Dockerfile
+  #     args:
+  #       MOFED_VERSION: 5.1-2.5.8.0
+  #       OS_VERSION: 12.5
+  #       CUDA_VERSION: 11.1
+  #       MOFED_OS: sles12sp5
+  # sles15.2-mofed5.1-cuda11.1:
+  #   image: sles12-mofed5.1-cuda11.1
+  #   build:
+  #     context: .
+  #     network: host
+  #     dockerfile: sles-release.Dockerfile
+  #     args:
+  #       MOFED_VERSION: 5.1-2.5.8.0
+  #       OS_VERSION: 15.2
+  #       CUDA_VERSION: 11.1
+  #       MOFED_OS: sles15sp2
```

Comment:
remove?

Reply:
It's question, sles don't have official docker container